Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb

```diff
@@ -0,0 +1,1498 @@
+{
```

Comment:
<p>Just a quick post merge fix (since you have made the PR not editable by maintainers), could you use this code to display the notebook buttons, a quick note these links will work well post merge:</p><p> </p><pre class="ql-syntax" spellcheck="false">&lt;table class="tfo-notebook-buttons" align="left"&gt;
  &lt;td&gt;
    &lt;a target="_blank" href="https://colab.research.google.com/github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Logistic%20Regression/Logistic_Regression.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/colab_logo_32px.png" /&gt;Run in Google Colab&lt;/a&gt;
  &lt;/td&gt;
  &lt;td&gt;
    &lt;a target="_blank" href="https://github.com/Rishit-dagli/Python3-by-practice/blob/master/Machine%20Learning/sklearn%20examples/Logistic%20Regression/Logistic_Regression.ipynb"&gt;&lt;img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" /&gt;View source on GitHub&lt;/a&gt;
  &lt;/td&gt;
&lt;/table&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I think the links, that I have used working are working.

Reply:
@Avinashshah099 The links you have used are working but to maintain consistency and ensure uptime (since you have linked to a drive file which could be deleted), I have included links to use Colab for GitHub and enforce those in all notebooks.

Reply:
wait, I am updating that.

Reply:
done, please have a look